recipes/ldas-tools-al-swig/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "ldas-tools-al-swig" %}
+{% set version = "2.6.4" %}
+{% set sha256 = "70c218dcdd000c9c6108b297e134b316a37d89fbd2627e1a47a459690c5a43d1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - make
+    - cmake >=3.6
+    - {{ compiler('cxx') }}
+  host:
+    - swig
+    - ldas-tools-al >=2.6.0
+    - ldas-tools-cmake >=1.0.3
+    - boost-cpp
+    - python
+  run:
+    - ldas-tools-al >=2.6.0
+    - python
+
+test:
+  imports:
+    - LDAStools
+
+outputs:
+  - name: python-ldas-tools-al
+    requirements:
+      host:
+        - ldas-tools-al >=2.6.0
+        - python
+        - numpy
+      run:
+        - ldas-tools-al >=2.6.0
+        - python
+        - {{ pin_compatible('numpy') }}
+    files:
+      - lib/python*/site-packages/LDAStools/__init__.py*
+      - lib/python*/site-packages/LDAStools/__pycache__/__init__.*.py*  # [not py27]
+    test:
+      imports:
+        - LDAStools
+    about:
+      home: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+      dev_url: https://git.ligo.org/ldastools/LDAS_Tools
+      doc_url: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+      license: GPLv2
+      license_family: GPL
+      license_file: COPYING
+      summary: Python bindings for the LDAS Tools abstraction toolkit
+
+about:
+  home: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+  doc_url: https://ldastools.docs.ligo.org/LDAS_Tools/Beta/doc/ldas-tools-al/ldastoolsal/html/
+  license: GPLv2
```

Comment:
It seems that you license is ``GPLv3``

Reply:
@marcelotrevisani, you are correct, thanks for the correction, I fixed this in a6a6db4.